tests/e2e/pageObjects/recommendations-page.ts

```diff
@@ -0,0 +1,33 @@
+import {Selector, t} from 'testcafe';
+
+export class RecommendationsPage {
+    veryUsefulVoteBtn = Selector('[data-testid=very-useful-vote-btn]').nth(0);
+    usefulVoteBtn = Selector('[data-testid=useful-vote-btn]').nth(0);
+    notUsefulVoteBtn = Selector('[data-testid=not-useful-vote-btn]').nth(0);
+    recommendationsFeedbackBtn = Selector('[data-testid=recommendation-feedback-btn]');
+
+    async voteForVeryUsefulAndVerifyDisabled(): Promise<void> {
+        await t.click(this.veryUsefulVoteBtn);
+        await this.verifyVoteDisabled();
+    }
+
+    async voteForUsefulAndVerifyDisabled(): Promise<void> {
+        await t.click(this.usefulVoteBtn);
+        await this.verifyVoteDisabled();
+    }
+
+    async voteForNotUsefulAndVerifyDisabled(): Promise<void> {
+        await t.click(this.notUsefulVoteBtn);
+        await this.verifyVoteDisabled();
+    }
+
+    async verifyVoteDisabled(): Promise<void>{
```

Comment:
we are trying to not use methods for expects into pageObject folder
for this we use tests\e2e\common-actions
please move this function to common-actions